public_suffix_list.dat

```diff
@@ -13270,6 +13270,10 @@ myshopify.com
 // Submitted by Craig McMahon <craig@shopitcommerce.com>
 shopitsite.com
 
+// Shoppoint : https://www.shoppoint.mo/
+// Submitted by David Ng <support@alphasolution.com.mo>
```

Comment:
Should this be a shoppoint.mo address?

Reply:
Yes, it is shoppoint.mo

Reply:
@simon-friedberger I don't get it, why is this PR getting closed?